src/parser/scanner.lex

```diff
@@ -124,6 +124,7 @@ LABEL_FULL_WIDTH            {CN_EN_FULL_WIDTH}{CN_EN_NUM_FULL_WIDTH}*
 "ADD"                       { return TokenType::KW_ADD; }
 "CREATE"                    { return TokenType::KW_CREATE;}
 "DROP"                      { return TokenType::KW_DROP; }
+"CLEAR"                     { return TokenType::KW_CLEAR; } 
```

Comment:
New reserved word means incompatible.

Reply:
Weird. Doesn't the incompatibility mean that the  this function affects other functions?

Reply:
Means user maybe need to modify their application.

Reply:
If the user does not use the clear space function, then there is no need to modify it. Add a new syntax,  I think it is forward compatible.

Reply:
When user has column of table named `clear`

Reply:
OK, I see.